extensions/amp-minute-media-player/0.1/test/test-amp-minute-media-player.js

```diff
@@ -75,7 +75,7 @@ describes.realWin(
       });
     });
 
-    it('renders with semantic (empty params)', () => {
+    it('renders with semantic (empty params)', async () => {
       return getMinuteMediaPlayer({
```

Comment:
Change `.then` to `await`.

Reply:
Changed

Reply:
Can you please change the `return X().then(...)` to an `await` pattern?

Reply:
(My bad, this is correct. I was looking at an older diff.)